README.md

```diff
@@ -1,2 +1,47 @@
 # ProjectBadges
+
 Badges for Formidable Open Source Projects
+
+## Contents
+
+- [Using the package](#using-the-package)
+- [Component props](#component-props)
+  - [Examples](#examples)
+- [Try component locally](#try-the-component-locally)
+
+## Using the package
+
+- `yarn add formidable-oss-badges` or `npm install formidable-oss-badges`
+- In your react app, `import { ProjectBadge } from "formidable-oss-badges;`
+- Use the `<ProjectBadge />` component as directed below
+
+## Component props
+The `<ProjectBadge />` component takes five, optional, props:
```

Comment:
Are they optional? There was a bit of back and forth on that. For optional values, you'd want to also list ther default value. For required props, you'd want to list them first and make a note that they're required.

Reply:
Yep all optional now, but have added a note about recommended props to the latest draft!